hrpsys_ros_bridge_tutorials/models/gen_sensor_attached_hrp2w_model.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
Because hrp2w has no legs I had to comment out appropriate lines else error results.